judge/templates/judge/submission_detail.html

```diff
@@ -145,17 +147,17 @@
                                         <div class="card shadow border-0">
                                             <div class="card-body px-lg-5 py-lg-5">
                                                 <div class="text-center mb-4">
-                                                    Test case
+                                                    Test Case {{ forloop.counter }}
```

Comment:
Why case starts with capital letter?

Reply:
Well, I thought it'd look better.

"Test Case" looks better than "Test case" :P

Reply:
For me, Test case makes more sense. Anyways your call.

Reply:
Let it be "Test Case".